superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -289,9 +289,10 @@ class TabbedSqlEditors extends React.PureComponent {
       const title = (
         <>
           {qe.title} <TabStatusIcon tabState={state} />{' '}
-          <span className="close" onClick={() => this.removeQueryEditor(qe)}>
-            {'×'}
-          </span>
+          <i
+            className="fa fa-close"
```

Comment:
Another option is to use `text-muted` here.

We have grays defined somewhere I think. We should make classes for our grays that we can use anywhere...

Reply:
https://github.com/apache/incubator-superset/blob/master/superset-frontend/stylesheets/less/variables.less#L48-L56

not sure how to use these grays in this context, @rusackas ?
- create classes in `superset.less` and use them as classes
- create a `./TabbedSqlEditors.less` and use the variable in there?
- use emotion?

Reply:
it might be even better to use emotion here. The colors should all be in `styled`

Reply:
thanks for suggestion. @rusackas do we have any example that i can follow to make similar thing? i am not sure which gray to use. thanks!